scripts/set-backend-env-vars.sh

```diff
@@ -0,0 +1,50 @@
+
+set -o allexport
+[[ -f ./tdrs-backend/.env.production ]] && source ./tdrs-backend/.env.production
```

Comment:
same q re: `.env.production`. want to be sure I understand where the keys are. I believe our goal is to leverage cloud.gov to get and set any secret keys, yes?

Reply:
This is only used if you're running the script locally - in the CI context it won't have that file and will skip sourcing the environment variables, instead using what is set in Circle CI project settings.

Reply:
@jtwillis92 @ADPennington I remember during Dev Sync yesterday we decided to change this line, but I don't remember how exactly we planned to change it ... possibly just remove it, since our preference would be not to store keys locally if possible?

Reply:
Ah yeah - we had discussed removing this and adding documentation to the README on how to get environment variables from cloud.gov if needed.

Reply:
Removed!